netd.yaml

```diff
@@ -162,7 +162,7 @@ spec:
       serviceAccountName: netd
       terminationGracePeriodSeconds: 0
       nodeSelector:
-        beta.kubernetes.io/kube-netd-ready: "true"
+        cloud.google.com/gke-netd-ready: "true"
```

Comment:
I thought thockin@ said something about gke.io instead of using cloud.google.com?

Reply:
node_startup.go is using cloud.google.com